packages/SwingSet/src/kernel/kernel.js

```diff
@@ -39,6 +39,9 @@ function makeError(s) {
   return harden({ body: JSON.stringify(s), slots: [] });
 }
 
+const VAT_TERMINATION_ERROR = makeError('vat terminated');
+const UNKNOWN_VAT_ERROR = makeError('unknown vat');
```

Comment:
I don't think userspace should ever be able to see this "unknown vat" error, and the fact that it exists makes me worried. They should only ever be able to see "vat terminated", and it shouldn't matter whether their message was early (pipelined to some promise that the kernel took over as executor of the dead vat's estate) or late (sent to an object-id whose owner is now "none"), they should always see the same error message.